Source/WebCore/page/Quirks.cpp

```diff
@@ -1974,5 +1974,15 @@ bool Quirks::shouldIgnoreContentObservationForClick(const Node& targetNode) cons
 
 #endif // PLATFORM(IOS_FAMILY)
 
+bool Quirks::needsMozillaFileTypeForDataTransfer() const
```

Comment:
This is missing the reference to a remove Quirk bug on bugs.webkit.org and/or the reference to the radar.

Reply:
Whoops — I'll fix that in a followup...